presto-base-jdbc/src/test/java/com/facebook/presto/plugin/jdbc/TestJdbcQueryBuilder.java

```diff
@@ -74,6 +76,8 @@
 
     private List<JdbcColumnHandle> columns;
 
+    private final CharType charType = CharType.createCharType(CharType.MAX_LENGTH);
```

Comment:
inlne, use some other length than max length (it has no special meaning here)

Reply:
Okay I was trying to not repeat myself, but if you want this in the couple of places I use it, I can do that.